gnutls-kdh-1.0.drv doesn't build

**Describe the bug**

See the [attached log](https://github.com/NixOS/nixpkgs/files/3776432/build.log); the relevant issue seems to be here:

```text
aes-padlock.c:96:29: error: 'struct aes_ctx' has no member named 'keys'; did you mean 'key_size'?
   memcpy(pce->ks.rd_key, nc.keys, sizeof(nc.keys));
                             ^~~~
                             key_size
aes-padlock.c:96:45: error: 'struct aes_ctx' has no member named 'keys'; did you mean 'key_size'?
   memcpy(pce->ks.rd_key, nc.keys, sizeof(nc.keys));
                                             ^~~~
                                             key_size
aes-padlock.c:97:22: error: 'struct aes_ctx' has no member named 'rounds'
   pce->ks.rounds = nc.rounds;
                      ^
```

Most of [my configuration](https://gitlab.com/victor-engmark/root/blob/b586ec1cf0606b9c558061b81020fde0b44533d2/configuration.nix) is online verbatim.

**To Reproduce**
Steps to reproduce the behavior:
1. `sudo nixos-rebuild switch --upgrade`

**Metadata**

 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.78, NixOS, 20.03pre196725.94500c93dc2 (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.1`
 - channels(root): `"nixos-20.03pre198214.4cd2cb43fb3"`
 - channels(username): `"nixpkgs-20.03pre193309.c4196cca9ac"`
 - nixpkgs: `/home/username/.nix-defexpr/channels/nixpkgs`

--- a/pkgs/applications/misc/lutris/chrootenv.nix
+++ b/pkgs/applications/misc/lutris/chrootenv.nix
@@ -29,7 +29,7 @@ in buildFHSUserEnv {
 
     # Dolphin
     bluez ffmpeg gettext portaudio wxGTK30 miniupnpc mbedtls lzo sfml gsm
-    wavpack gnutls-kdh orc nettle gmp pcre vulkan-loader
+    wavpack orc nettle gmp pcre vulkan-loader
 
     # DOSBox
     SDL_net SDL_sound

